tests/dummy/app/controllers/application.js

```diff
@@ -5,7 +5,7 @@ export default class ApplicationController extends Controller {
   plugins = [{
     name:'insert-variable',
     options: {
-      administrativeUnitUuid: 'cec59e5e872a9084e93becf3026bfcc2f25926ea76372711b7a745875f3b7949'
+      publisher: 'cec59e5e872a9084e93becf3026bfcc2f25926ea76372711b7a745875f3b7949'
```

Comment:
wouldn't it make more sense if this was a URI?

Reply:
<img width="627" alt="Screen Shot 2022-09-14 at 12 23 07" src="https://user-images.githubusercontent.com/17006446/190129629-83eab40a-f210-48ba-af12-e7d2bd8a5f6b.png">
The problem is on GN with the current session service I only get the id but not the uri, I didn't check why it could be a bug with the service

Reply:
but you could still request the URI? our identifiers are URIs so we should use those